src/app/complex/complex-results/complex-list/complex-list.component.html

```diff
@@ -1,3 +1,34 @@
+<div class="Complex-navigator">
+  <table id="CN-table">
+    <thead>
+    <tr>
+      <th>Components/complexes</th>
+      <th *ngFor="let complex of complexSearch.elements">
+        <div class="complexInTable">
+          <a [routerLink]="['/complex', complex.complexAC]">
+            {{complex.complexAC}}
+            <span class="ComplexNameSpan">{{complex.complexName}}</span>
+          </a>
+        </div>
+      </th>
+    </tr>
+    </thead>
+    <tbody *ngFor="let complex of complexSearch.elements">
+    <tr *ngFor="let component of complex.componentIds">
+      <td>{{component}}</td>
+      <td *ngFor="let complex of complexSearch.elements">
+        <ng-container *ngFor="let component of complex.componentIds">
+          x
+        </ng-container>
```

Comment:
We need to replace this with a call to the new method that checks if a component in
```
<div *ngIf="doesComplexContainComponent(complex, component)">X</div>
```